runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/runtime/ILog.java

```diff
@@ -132,4 +134,45 @@ default void error(String message) {
 	default void error(String message, Throwable throwable) {
 		log(new Status(IStatus.ERROR, getBundle().getSymbolicName(), message, throwable));
 	}
+
```

Comment:
Not it that it really matters, but I have the impression that usually such static factories methods are added at the top of the file instead of the bottom.